Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -386,6 +386,11 @@ extension PlaygroundController {
             self.clientSecret = json["intentClientSecret"]
             self.ephemeralKey = json["customerEphemeralKeySecret"]
             self.customerID = json["customerId"]
+            print(self.clientSecret)
```

Comment:
Lets remove the prints

Reply:
oh whoops! this was some testing for an unrelated run task